google-cloud-storage/samples/storage_upload_file.rb

```diff
@@ -26,7 +26,7 @@ def upload_file bucket_name:, local_file_path:, file_name: nil
   require "google/cloud/storage"
 
   storage = Google::Cloud::Storage.new
-  bucket  = storage.bucket bucket_name
+  bucket  = storage.bucket "bucket_name", skip_lookup: true
```

Comment:
Please remove the quotes around `bucket_name`. We're passing a variable, not a literal string.

Reply:
Once you make this change, LGTM, thanks for addressing this @rebecca-pete!!